libdevcore/JSON.h

```diff
@@ -23,22 +23,97 @@
 #pragma once
 
 #include <json/json.h>
+#include <sstream>
+#include <memory>
 
 namespace dev
 {
 
+namespace json
+{
+
+/// StreamWriterBuilder that can be constructed with specific settings
+class StreamWriterBuilder : public Json::StreamWriterBuilder {
+public:
+	explicit StreamWriterBuilder(const std::map<std::string, std::string> &_settings)
+	{
+		for (auto& iter : _settings)
+		{
+			this->settings_[iter.first] = iter.second;
+		}
+	}
+};
+
+/// CharReaderBuilder with strict-mode settings
+class StrictModeCharReaderBuilder : public Json::CharReaderBuilder {
```

Comment:
I think those classes are better suited in an anonymous namespace in the `.cpp` file - they are only used by the functions in this header, right?

Reply:
Yep, currently it's just used by the functions in header. I thought that it could make sense to allow others to use those classes. But I think that will probably never used, because we can already parse and print `JSON` objects, that are the most important operations. I moved it to the `.cpp` file.